samples/edx/dashboard.html

```diff
@@ -28,9 +28,9 @@
 
             <h2 class="mb-4">Courses</h2>
 
-            <h3>In-Progress</h3>
+            <h4>In-Progress</h4>
```

Comment:
I'm just looking at this because I'm curious to know about the bootstrap work.  
Wondering why we are skipping a header level here?

Reply:
The difference wasn't clear enough from a look-and-feel perspective. As long as there are no more h3's under the h4 then the screen reader will pick it up as normal and we don't have to write custom css.

Reply:
Interesting.  Seems like a styling issue if we can't have a doc with h1-3 and have it look meaningful, but maybe not?

Reply:
@robrap @HarryRein It is an a11y violation to have an h4 beneath an h2 like this. As @robrap says, we should ensure that the styling looks right, rather than changing the DOM structure.

I can't find our own document on this, but here's the W3C:

https://www.w3.org/WAI/tutorials/page-structure/headings/